Source/DafnyRuntime/DafnyRuntime.cs

```diff
@@ -1,12 +1,12 @@
+//-----------------------------------------------------------------------------
+//
 // Copyright by the contributors to the Dafny Project
 // SPDX-License-Identifier: MIT
-
-#if ISDAFNYRUNTIMELIB
```

Comment:
Was removing this conditional compilation intentional?

Reply:
Yes, this was part of the removing the formatting annotations.There was warning "Using directive should appear within a namespace declaration." So I have added both "using System" and "using System.Numerics" in both namespaces and removed from there.

Reply:
Moving it is fine, but may still need the `#if ISDAFNYRUNTIMELIB` guard. This is because this file is used in two different ways: both as source for the `DafnyRuntime` project (and the `csproj` file sets `ISDAFNYRUNTIMELIB` to true), but also as raw text that the C# compiler will prepend to the compiled program output (in which case `ISDAFNYRUNTIMELIB` is false by default).

I'm not actually convinced it IS still necessary, mind you. But we need to double check both use cases work, and I'm a little worried the test suite doesn't cover this well. You might want to detach this improvement from this PR for the sake of avoiding scope creep.

Reply:
okay, I am doing that. That can be addressed later.